eng/scripts/Automation-Sdk-Generate.ps1

```diff
@@ -1,6 +1,12 @@
+[CmdletBinding()]
 param(
+  [Parameter(Mandatory = $True)]
   [string] $InputJsonPath = $GenerateInput,
```

Comment:
1. To @pakrym's question, where are these variables coming from?
2. Putting `Mandatory=$true` on this will require a user still pass a value on the command line. If you want to default it but specify a value is required, so this instead:

```powershell
[Parameter()]
[ValidateNotNullOrEmpty()]
[string] $InputJsonPath = $GenerateInput,
```

This way, if `$GenerateInput` is null or empty (same thing in PowerShell by default in most cases), it'll prompt the user in interactive mode or fail early in non-interactive modes (like during build).

Reply:
As I mentioned [here](https://github.com/Azure/azure-sdk-for-net/pull/17193#discussion_r533644697) 
>[GenerateInput ](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=107) and [GenerateOutput](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=270) are passed in [SDK Automation step](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4254356&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5). It is configured in the pipeline.

Updated!
>[Parameter()]
[ValidateNotNullOrEmpty()]
[string] $InputJsonPath = $GenerateInput

Reply:
I don't think you need the ` = $GenerateInput` part. You would get the parameter passed into the script without it.

Reply:
Yes, you're right. 

Updated!